assembl/graphql/idea.py

```diff
@@ -785,6 +787,16 @@ def create_idea(parent_idea, phase, args, context):
     return saobj
 
 
+def tombstone_posts_related_to_idea(idea, context):
+    error = _("Cannot delete posts related to the idea because no idea was found.")
+    if idea is None:
+        raise Exception(context.localizer.translate(error))
```

Comment:
Why do this? We are sure that idea is not None here, you call it with thematic, if it was None, the code would crash before the call.